src/templates/main.php

```diff
@@ -20,10 +20,15 @@
 <?php endif;?>
     </path>
     <text font-family='"<?php echo $font ?>", monospace' fill='<?php echo $color ?>' font-size='<?php echo $size ?>'
+<?php if ($vCenter): ?>
+        dominant-baseline='middle'
+<?php else: ?>
+        dominant-baseline='auto'
```

Comment:
Technically speaking, dominant-baseline="auto" doesn't put it at the top, but rather places letters on top of the middle line instead of centering the text with the middle line.

To make it actually top aligned, we could make `dominant-baseline="hanging"` and have the y coord be `0` instead of `$height / 2` (see line 14)

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/dominant-baseline

Reply:
I would also argue, that it is good like that, so that it doesn't change positioning for the existing setups, like you mentioned.

What we could do, is adjust the documentation about it. Maybe to something like this: `vCenter: true to center text vertically or false to place it on top of the middle line (default: false)` 
(Happy about any wording suggestions as I'm not a native speaker)

Reply:
@8BitJonny Sounds good how about 

> `true` to center vertically or `false` to align above the center

We should try to keep it short so it doesn't make the table stretch too much (although not a huge issue)

|  Parameter  |                                 Details                                  |  Type   |               Example               |
| :---------: | :----------------------------------------------------------------------: | :-----: | :---------------------------------: |
|  `vCenter`  |     `true` to center vertically or `false` to align above the center     | boolean |          `true` or `false`          |

Reply:
changed 👍🏼